DetectDynamicJS.py

```diff
@@ -77,6 +77,11 @@ def doPassiveScan(self, baseRequestResponse):
         statusCode = responseInfo.getStatusCode()
         if statusCode != 200:
           return
+        
+        # Check if the script is world readable
```

Comment:
As I said previously, browsers cannot read x-domain resources or scripts (in most cases) if they respond with status code other than 200. Please try it yourself :)